tests/suites/test_suite_ssl.function

```diff
@@ -208,12 +219,19 @@ int mbedtls_test_message_queue_push_info( mbedtls_test_message_queue *queue,
  * the oldest inserted message length(fifo). Additionally, available buffer
  * length will be checked to ensure that the message can fit into it (so that
  * the metadata will stay in the queue in case of an insufficient buffer).
+ *
+ * \retval  MBEDTLS_TEST_ERROR_ARG_NULL, if the queue or msg_len is null.
+ * \retval  MBEDTLS_TEST_ERROR_QUEUE_EMPTY, if the queue is empty.
+ * \retval  MBEDTLS_TEST_ERROR_BUFFER_TOO_SMALL, if the given buffer length is
+ *          too small to fit the message.
+ * \retval  0, if the pop was succesfull, and the message will fit in a given
+ *          buffer length.
```

Comment:
This should be added in "Add a message metadata queue in ssl tests", not "Add a message-based socket mock connection to the ssl tests"